### ISSUE TITLE
Add 'context' attribute to <element> tag

### ISSUE THREAD

**Issue by gregjacobs** [xframium/xframium-java#21]
Hey guys, was just talking to @allengeary and wanted to record a possible suggestion for dealing with native/webview context in a hybrid app: add a 'context' attribute to the `<element>` tag.

An example:

``` xml
<model siteName="My App">
    <page name="Login">
        <element name="USERNAME" value=".username" descriptor="CSS" context="WEBVIEW_1" />
        <element name="PASSWORD" value=".password" descriptor="CSS" context="WEBVIEW_1" />
        <element name="SUBMIT" value=".submit" descriptor="CSS" context="WEBVIEW_1" />
        <element name="LOCATION_ALERT_ALLOW_BTN" value="//UIAButton[@name='Allow']" descriptor="XPATH" context="NATIVE" />
    </page>
</model>
```

This would save developers from needing to add manual steps to tests to switch between contexts.

**Comment by xframium** [comment]
This is in progress but did not make the 1.0.5 build.  It will be released in 1.0.6

**Comment by xframium** [comment]
devicContext is now an attribute on the element tag automatically switching to the specified context of the underlying web driver is context aware
